resources/assets/uad_lists.json

```
@@ -37236,5 +37236,59 @@
     "dependencies": [],
     "neededBy": [],
     "labels": []
+  },
+  {
+    "id": "com.android.DeviceAsWebcam",
+    "list": "Oem",
+    "description": "Allows using certain Pixel phones as a USB webcam.\n Will break USB function picker.",
```

```suggestion
    "description": "Allows using certain Pixel phones as a USB webcam.\nWill break USB function picker.",
```
Remove redundant space.